docs/releases/v1.1.0.rst

```diff
@@ -97,6 +101,9 @@
     -[fix] Reducers and row-wise functions now support :attr:`void <dt.Type.void>`
        columns. [#3284]
 
+    -[fix] All the row-wise functions now support :attr:`void <dt.Type.void>`
+       columns. [#3284]
+
```

Comment:
This should be removed as we already mention `row-wise functions` above. Probably this piece of code is left from one of the old merges from main.